README.md

```diff
@@ -22,18 +18,146 @@ All `.md` files within the `src/` folder will be treated as something that relat
+
+# Static assets
+
+GitHub is used to serve the static assets shown on Zendesk. These assets are located under the `static` folder at the root of this repo. When referencing static assets, use them as follows:
+
+```markdown
+For an image:
+![Clip showing the 3 LEDS on Home Assistant Green](/static/img/green/green_yellow_led_heartbeat.webp)
+For a link:
+[Datasheet](/static/docs/connect-zbt-1/Home-Assistant-Connect-ZBT-1_Datasheet_v2_0.pdf)
+```
+
+Upon build, Eleventy will replace all the static assets with the correct URL for the asset. This allows for easy referencing of static assets in the content.
+
+# Internal linking
 
-TODO: Update this section
+If you need to link to other articles within Zendesk, use a relative URL without the domain. For example, if I want to link to this article: https://support.nabucasa.com/hc/en-us/articles/24737667232413-Setting-up-the-device, I would use the following markdown:
 
-- Clone this repo into a docker container volume
-- Create a `.env` file at the root of the project, copying the contents of `.env.example`, replacing the values with your own
-- Run `npm run start` to start the Zendesk theme compiler and the preview server
+```markdown
+Correct link:
+[Setting up the device](/hc/en-us/articles/24737667232413)
+Incorrect link (using slug after the article ID):
+[Setting up the device](/hc/en-us/articles/24737667232413-Setting-up-the-device)
+Incorrect link (missing starting slash in URL):
+[Setting up the device](hc/en-us/articles/24737667232413)
+Incorrect link (using URL domain):
+[Setting up the device](https://support.nabucasa.com/hc/en-us/articles/24737667232413)
+```
 
-## Content preview
+Notice how the slug of the article is not included in the link. (e.g. use `/hc/en-us/articles/24737667232413` instead of `/hc/en-us/articles/24737667232413-Setting-up-the-device`). This is because Zendesk uses the title of the object as the slug which is subject to change with future updates. Conveniently, going to the URL without the slug will redirect to the correct URL with the slug. This will allow for easy linking to other articles without worrying about the slug changing in the future.
 
-To ease content creation, a preview is available. Keep in mind that is just a preview to help visualize content, some features are not working (search engine for example). To run it:
+# How to contribute
 
-- open the devcontainer in Visual Studio Code
-- the task `Content preview` (or run `npm run start`)
-- visit `http://127.0.0.1:8080`
+- Clone this repo into a docker container volume.
+- Create a `.env` file at the root of the project, copying the contents of `.env.example`, replacing the values with your own.
```

Comment:
No contributor should ever need this .env file, ideally the file itself would be removed as this is only used by the github action and should never be run manually.